readme.md

```diff
@@ -563,6 +563,53 @@ getRootInjector().register(Car, { resolution: SuperCar });
 
 In the section under **Global Configuration** you can learn about how you can use **External Resolution Strategies** to delegate construction globally and provide fallback strategies when performing interop with other container systems. 
 
+# Register Value
+
+JavaScript has a number of intrinsic types that you may wish to injector directly into a constructor without the need for wrapping in a higher type. These types include `Array`, `Boolean`, `Date`, `Error`, `Function`, `JSON`, `Number`, `RegExp` and `String`. In order to support direct injection of these types we can use the `registerValue` method found on the injector.  The `registerValue` method only takes one parameter, the injection configuration object. There are two ways to resolve the value, you can either provide a value at point of registration or you can use a resolution strategy to resolve at point of use.  Below are some examples to illustrate how this works. 
+
+## Registering with a AOT value
```

Comment:
not sure we should be using AOT / JIT to explain this. Makes me (and I suspect others) think of angular. Happy to ignore if you want to continue using.

Reply:
Any recommendations welcome